### ISSUE TITLE
Question about Tab.5 which compares different pseudo labeling strategies

### ISSUE THREAD

**Issue by LiheYoung** [googleinterns/wss#4]
Thanks for your impressive work!

I notice that in Tab.5, different pseudo labeling strategies are compared. Whether the numbers listed here are mIOU on validation set or mIOU of pseudo masks on unlabeled images from the training set? 

It seems the latter one, if so, are these unlabeled images from the VOC original training set(1,464 images) or the augmented SBD images?

**Comment by Yuliang-Zou** [comment]
It is actually the former one. These mIoUs are on the validation set.

**Comment by LiheYoung** [comment]
But it is somewhat strange that the result(73.79) in Tab.5 of 1/4 labeled setting are higher than that(73.23) in Tab.1 of 1.4k labeled setting? 

Is there anything I misunderstood?

**Comment by Yuliang-Zou** [comment]
That's because they are using different network architectures (and actually different pre-training schedules). In Table 5 we use Xception-65, while in Table 1 we use ResNet. You can find in the original DeepLabv3+ paper that they tune the ImageNet pre-training strategy a bit to get a strong performance for Xception-65.

**Comment by LiheYoung** [comment]
I got it! Thank you very much!

By the way, are all these results produced with single-scale testing, rather than multi-scale testing and flipping?

**Comment by Yuliang-Zou** [comment]
Yes. All results are tested using a single-scale only, without any test-time augmentations.

**Comment by LiheYoung** [comment]
Thanks for your prompt reply!
